src/Domain/MoveValidator.cs

```diff
@@ -8,36 +9,142 @@
 
 namespace UltimateTicTacToe.Domain
 {
+    internal class BoardWinnerEvaluator
+    {
+        private readonly TileInformation[,] _tiles;
+
+        public BoardWinnerEvaluator(TileInformation[,] tiles)
+        {
+            _tiles = tiles;
+        }
+
+        internal Winner GetWinner(Player lastPlayer)
+        {
+            // TODO: implement
+
+            // has LastPlayer won ==> Wiiner = Playser
+            // is Board full ==> Winner ==> Draw
+            // Winner ==> none
+        }
+
+        private bool IsBoardFull()
+        {
+            for (int x = 0; x < 3; x++)
+            {
+                for (int y = 0; y < 3; y++)
+                {
+                    if (_tiles[x, y].Value == TileValue.Empty)
```

Comment:
Use list.Any instead?

Reply:
The any extension is not available because _tiles is a multidimensional Array. In my opinion it's the only downside of using this type.

Reply:
Switched Type. Any can be used now